app/presenters/tree_builder_buttons.rb

```diff
@@ -35,18 +43,4 @@ def x_get_tree_custom_kids(object, count_only, _options)
     )
     count_only_or_objects(count_only, objects)
   end
-
-  def buttons_node_image(node)
-    case node
-    when 'ExtManagementSystem' then {:icon  => 'pficon pficon-server'}
-    when 'CloudTenant'         then {:icon  => 'pficon-cloud-tenant'}
-    when 'EmsCluster'          then {:icon  => 'pficon pficon-cluster'}
-    when 'Host'                then {:icon  => 'pficon pficon-screen'}
-    when 'MiqTemplate'         then {:icon  => 'ff ff-template'}
```

Comment:
@skateman @himdel I think we should still be using ff-template for... templates.